packages/react-renderer-demo/src/app/src/components/docsearch.js

```diff
@@ -54,7 +54,7 @@ const DocSearch = () => {
       handleSelected: (input, event, suggestion) => {
         event.button = 0;
         const resultUrl = new URL(suggestion.url);
-        push(resultUrl.pathname);
+        push(`${resultUrl.pathname}${resultUrl.hash}`);
```

Comment:
what if the hash is undefined?

Reply:
Can it be?

Reply:
Well thet depends on the algolia crawler. We are not in control here